modules/storage-rocksdb/src/test/java/org/apache/ignite/internal/storage/rocksdb/RocksDbTableStorageTest.java

```diff
@@ -246,7 +248,8 @@ void storageAdvertisesItIsPersistent() {
         assertThat(storage.isVolatile(), is(false));
     }
 
-    private IgniteBiTuple<TestKey, TestValue> unwrap(BinaryRow binaryRow) {
+    @Nullable
+    private static IgniteBiTuple<TestKey, TestValue> unwrap(@Nullable BinaryRow binaryRow) {
```

Comment:
```suggestion
    private static @Nullable IgniteBiTuple<TestKey, TestValue> unwrap(@Nullable BinaryRow binaryRow) {
```

Reply:
Why?

Reply:
This is just a suggestion: null can be a return value, not a modifier.